.dockerignore

```diff
@@ -1,8 +1,8 @@
 *
+
```

Comment:
I removed Gemfile.lock from this to do not include it during docker building process, this causes issues when running bundle install over a Gemfile.lock copied from local to the container that was generating using a different ruby version